core/services/ardupilot_manager/mavlink_proxy/Manager.py

```diff
@@ -32,11 +36,63 @@ def available_interfaces() -> List[Type[AbstractRouter]]:
     def use(self, interface: AbstractRouter) -> None:
         self.tool = interface
 
+    def _saved_endpoints(self) -> List[Endpoint]:
+        try:
+            with open(self.endpoints_path) as f:
+                return [Endpoint(dictionary=endpoint) for endpoint in json.load(f)]
```

Comment:
Why the manager is doing this ? This should be the settings job.

Reply:
I considered that the Settings job would be to provide the correct paths to the modules, not to perform the IO operations, but this can be revisited.

Do you think this is Setting's job or there should be a dedicated class for IO (it can be in the same file)?

Reply:
Right now it's on the Settings class, but we can segregate it. I'm thinking if it makes sense to.